index.html

```diff
@@ -89,44 +143,166 @@
 <script>
   var data = [];
   var options = {
-    yearRange: 'all'
+    categoryRange: 'all'
   };
+  
+
+  var svg, x, y;
 
-  var svg;
+  var margin = { top: 15, right: 15, bottom: 30, left: 45 } ; //spacing b/w svg and g
+
+  var width = 840 - margin.right - margin.left;
+  var height = 500 - margin.top - margin.bottom;
 
   // FETCH DATA
 
+  d3.json('school_data.json', function(error, json) {
+    if (error) { throw json;} //if error is defined, then don't do anything
+
+
+    
+
+    data = json;
+
+    setup(); //must occur after fetching the data
+    update();
+  });
+
   function setup() {
     // SVG
+   
 
     svg = d3.select('#chart')
-      .append('svg');
+      .append('svg')
+      .attr('width', width + margin.right + margin.left)
+      .attr('height', height + margin.top + margin.bottom)
+      .append('g')
+      .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
 
+
+ 
     // SCALES
 
+    var padding = 20;
+
+    x = d3.scale.linear()
+      .domain([0, d3.max(data, function (d) { return d.Reading_proficient/100; })]) //finds highest value of reading scores
+      .range([padding, width-margin.right])
+      .nice();
+
+
+    y = d3.scale.linear()
+      .domain(d3.extent(data, function (d) { return d.Math_proficient/100; })) 
+      .range([height-padding, margin.top])
+      .nice();
+
     // AXES
 
+    var formatPercentage = d3.format("1%");
+
+    var xAxis = d3.svg.axis()
+        .scale(x)
+        .orient("top")
+        .tickSize(height-padding-(margin.top*1.1)) //if i omit multiplying by 1.1, the lines go past the graph for some reason
```

Comment:
Once we've cleaned up the scales, this should be simpler. The issue is that we've set our scale to be an adjusted `height`. I'd expect just `height` to work here.